pages/docs-comissao.vue

```diff
@@ -2,43 +2,42 @@
-          <p><a href="https://drive.google.com/open?id=1UYVy424qBDmbzGkVMF3tHJLa-pURHA2I" target="_blank">Relação de convidados para a audiência pública</a></p>
-          <p><a href="https://drive.google.com/open?id=1MzKr_RXOjYriedVIp2dl-mQWJjCK6LgE" target="_blank">Requerimento n 27 Comissão de Educação, Cultura e Esporte</a></p>
-          <p><a href="https://drive.google.com/open?id=1SJh9vZ1hwfJcDWiJSi7rMlIM8Sevv6HJ" target="_blank">Solicitação do Tutor Alexandre Monteiro</a></p>
-          <p><a href="https://drive.google.com/open?id=1M3kaFqHnKttyX48S0s87wK4SkzYom4Dg" target="_blank">Abaixo Assinado para que a permanencia ou saida de tutores seja mediada por processo avaliativo</a></p>
-          <p><a href="https://drive.google.com/open?id=1uRzltpgpy1WFmqgh7mBqiK5Z3Iij7kJu" target="_blank">Relatório de Atividades da CENAPET referente ao período de 01/08/2014 a 20/06/2015 para análise por parte do Conselho- Documento encaminhado para o Conselho em 24/06/2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1NNjRlFksomuUDKoGwJ2YMTVrFZw0T0VV" target="_blank">Ata da 2ª Reunião da Diretoria da CENAPET com representantes da SESu/MEC em 2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1TNq8LNqCNhxJFQmk4hKQstrxcgET7FpG" target="_blank">Ata da 2a Reunião da CENAPET em 2015, realizada no dia 08/04/2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1DgblUOugGD1_xk-h0icniayt4DWNKYui" target="_blank">Ofício encaminhado pela CENAPET ao coordenador do NORTEPET 2015 no dia 08/04/2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1BtwLgdjBFibASs_0R0MnXr23RTsjBf6X" target="_blank">Ata da 1a Reunião da CENAPET em 2015, realizada no dia 25/03/2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1x7c0AOBYKwnB53i9Jf-Wo72GI4-wJ1AR" target="_blank">Ofício encaminhado pela Diretoria da CENAPET à SESu/MEC no dia 25/03/2015</a></p>
-          <p><a href="https://drive.google.com/open?id=1qOt2ecfb5yqhzCcGUSbVL7b0pIDFml8a" target="_blank">Ofício encaminhado pela Diretoria da CENAPET à SESu/MEC no dia 30/01/2015 reivindicando a regularização no pagamento das bolsas</a></p>
-        </div>
-      </div>
+    <div class="row no-gutters text">
+      <div v-html="$md.render(doccomissao.conteudo)"></div>
     </div>
   </div>
 </template>
 
+<script>
+import axios from 'axios';
+var moment = require("moment");
+
+export default {
+  data() {
+    return {
+      doccomissao: {},
+      moment: moment
+    };
+  },
+  asyncData ({ params }) {
+    return axios.get(`http://localhost:1337/Doc-Comissao/`)
+      .then((res) => {
+        return { doccomissao: res.data }
+      })
+  },
+  methods: {
+    // método para retornar a imagem de fundo padrão caso não haja nenhuma
+    backgroundURL: function() {
+      return this.noticia.img_fundo
+        ? process.env.baseURL + this.noticia.img_fundo.url
+        : require("~/assets/images/fundo.png");
+    }
```

Comment:
Esse método é só da página de notícias, pra verificar se a notícia tem uma imagem de fundo ou não. Pode tirar ele.